net-im/qtox/qtox-1.15.0.ebuild

```diff
@@ -0,0 +1,79 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils gnome2-utils xdg-utils
+
+MY_P="qTox-${PV}"
+DESCRIPTION="Most feature-rich GUI for net-libs/tox using Qt5"
+HOMEPAGE="https://github.com/qTox/qTox"
+SRC_URI="https://github.com/qTox/qTox/archives/v${PV}.tar.gz -> ${MY_P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="gtk test X"
+
+S="${WORKDIR}/${MY_P}"
+
+RDEPEND="
+	dev-db/sqlcipher
+	dev-libs/libsodium
+	dev-qt/qtconcurrent:5
+	dev-qt/qtcore:5
+	dev-qt/qtgui:5[gif,jpeg,png,xcb]
+	dev-qt/qtnetwork:5
+	dev-qt/qtopengl:5
+	dev-qt/qtsql:5
+	dev-qt/qtsvg:5
+	dev-qt/qtwidgets:5
+	dev-qt/qtxml:5
+	media-gfx/qrencode
+	media-libs/openal
+	>=media-video/ffmpeg-2.6.3[webp,v4l]
+	gtk? (	dev-libs/atk
+			dev-libs/glib:2
+			x11-libs/gdk-pixbuf[X]
+			x11-libs/gtk+:2
+			x11-libs/cairo[X]
+			x11-libs/pango[X] )
+	net-libs/tox:0/0.2[av]
```

Comment:
The minimum toxcore version seems to still be “0.1.*” for qTox, and I just confirmed that it still works with 0.1.10.

That support is probably going away soon(?) though, and it would certainly be better to use the latest version instead.  Just figured I'd at least make a mention of this.  :]